src/App.tsx

```diff
@@ -8,7 +8,39 @@
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
+import { СompletedStatus } from './types/СompletedStatus';
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const [query, setQuery] = useState<string>('');
+  const [selectedTodoId, setSelectedTodoId] = useState<number | null>(null);
+  const [filterStatus, setFilterStatus] = useState<СompletedStatus>(
+    СompletedStatus.all,
+  );
+
+  let flag = null;
```

Comment:
The variable `flag` is declared with `let` but is not reassigned. Consider using `const` instead for better code clarity and to prevent accidental reassignment.